recipe/meta.yaml

```diff
@@ -21,18 +21,18 @@ requirements:
     run:
         - python
         - pandas
+        - six
+        - gdal >=2.*
         - pyproj
         - shapely
         - fiona
-        - gdal 1.11.4
-        - six
         - rtree
-        - descartes
-        - matplotlib
-        - psycopg2  # [not win]
-        - sqlalchemy
-        - geopy ==1.10.0
+        - geopy 1.10.0
         - pysal
+        - matplotlib  # [not win and py35]
```

Comment:
yes, that sounds like a good idea!

What is actually the problem with matplotlib? (as the matplotlib builds in their feedstock seem fine)